test/unit/services/url.ts

```diff
@@ -636,6 +654,24 @@ suite('URL Service', ({ expect, spy, stub, itShouldBeCore, itShouldExtendBaseSer
       expect(filterState).to.be.calledWithExactly(state);
       expect(pushState).to.be.calledWith({ url, state: { data }, app: STOREFRONT_APP_ID }, '', url);
     });
+
+    it('should handle errors thrown by `pushState`', () => {
+      const err = new Error('Whoops, something went wrong!');
+      const pushState = stub().throws(err);
+      const warn = spy();
+      win.history = { pushState };
+      stub(service, 'filterState');
+      service['app'] = <any>{
```

Comment:
This is short enough to be on one line. Make sure to remove the comma when you do that.